Sources/BugsnagPerformance/Private/IdGenerator.h

```diff
@@ -17,15 +17,15 @@ namespace bugsnag {
 class IdGenerator {
 public:
     static SpanId generateSpanId() noexcept {
-        return random<SpanId>();
```

Comment:
Yes, the static analysis trips over this because it think's I'm calling `rand()` unless I give it a different name...